gulpfile.js

```diff
@@ -248,5 +255,5 @@ gulp.task('sync-meta', function() {
   appPackageJson.description = packageJson.description;
   appPackageJson.author = packageJson.author;
   appPackageJson.license = packageJson.license;
-  fs.writeFileSync('./src/package.json', JSON.stringify(appPackageJson, null, '  ') + '\n');
-});
+  fs.writeFileSync('./src/package.json', JSON.stringify(appPackageJson, null, '  '));
```

Comment:
This fixes './src/package.json' changing each time you run prettify or build. As build will add this '\n' and prettify will remove it.